src/OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.cpp

```diff
@@ -23,6 +23,61 @@ namespace gravitational
 
 using GeographicLib::GravityModel;
 
+using ostk::physics::units::Derived;
+using ostk::physics::units::Length;
+using ostk::physics::units::Time;
+using SphericalGravitationalModel = ostk::physics::environment::gravitational::Spherical;
+
+// EGM2008
+
+const Model::Parameters Earth::EGM2008 = {
+    {398600441500000.0, GravitationalParameterSIUnit},
+    Length::Meters(6378137.0),
+    1.0 / 298.257223563,
+    -4.84169317366974e-04,
+    5.39965866638991e-07};
```

Comment:
The formatting of this is quite weird... I wonder if we could tweak clang...

Reply:
We can! I'll add that trailing comma that I preached about on slack but didn't use in my own MR 🤦